### ISSUE TITLE
Add Comments to Entries

### ISSUE THREAD

**Issue by arnonrdp** [globe-and-citizen/Celebrity-Fanalyzer#37]
Unlike Prompts, Entries will have comments.

The idea is that the comments are nested (we need to decide which maximum level of depth). For that each subcomment can have a `parent_id` which is the `id` of the superior comment. The topmost comments can have a `parent_id = null`.

In addition to the comment component, you will also need a specific Pinia Store for comments, and it must post to a Firestore collection.

For this issue, it is not necessary to include upvotes/downvotes in comments. We can leave that task for after this one is ready.

Mock:
<img width="353" alt="Captura de Tela 2022-12-06 às 18 40 11" src="https://user-images.githubusercontent.com/26576201/206029026-bafd2bcf-b6ae-42e2-8630-d149efc96c38.png">

**Comment by stravid87** [comment]
Something I noticed -- that I really liked -- from Reddit, is that they employed an "infinite" nesting structure. As you go deeper and deeper, the size of the left margin is asymptotic. Really, "infinite" is not necessary but I think we could choose a large number like 10 levels. Making the css work might not be too tough. Just a matter of choosing a left margin set to a log(x) function that increases towards a limit. 

I looked at several other apps that used different pages beyond a depth of 2-3 levels and this made it very hard to follow.
